tests/10-issue.js

```diff
@@ -2,119 +2,260 @@
-        let issuerResponse;
-        let err;
-        let issuedVc;
-        before(async function() {
-          const issuer = issuers.find(issuer => issuer.tags.has('Suspension'));
-          const {result, error, data} = await issueVc({issuer});
-          err = error;
-          issuerResponse = result;
-          issuedVc = data;
-        });
-        it('MUST issue a VC with BitstringStatusList "credentialStatus" and ' +
-          '"suspension" status purpose', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          should.exist(issuerResponse);
-          should.not.exist(err);
-          issuerResponse.status.should.equal(201);
-          should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
-          testCredential({credential: issuedVc});
+          it('The purpose of the status entry MUST be a string.',
+            async function () {
+              this.test.cell = { columnId: issuerName, rowId: this.test.title };
+              issuedVc.credentialStatus.should.contain.keys('statusPurpose');
+              issuedVc.credentialStatus.statusPurpose.should.be.a('string');
+            });
+          it('While the value of the string is arbitrary, ' +
+            'the following values MUST be used for their intended purpose',
+            async function () {
+              if (issuedVc.credentialStatus.statusPurpose == 'revocation') {
+                this.test.cell = { columnId: issuerName, rowId: this.test.title };
+              } else if (issuedVc.credentialStatus.statusPurpose === 'suspension') {
+                this.test.cell = { columnId: issuerName, rowId: this.test.title };
+              } else if (issuedVc.credentialStatus.statusPurpose === 'message') {
+                this.test.cell = { columnId: issuerName, rowId: this.test.title };
+              } else {
+                this.skip()
+              };
+            });
+          it('The statusListIndex property MUST be an arbitrary size integer ' +
+            'greater than or equal to 0, expressed as a string in base 10.',
+            async function () {
+              this.test.cell = { columnId: issuerName, rowId: this.test.title };
+              issuedVc.credentialStatus.should.contain.keys('statusListIndex');
+              issuedVc.credentialStatus.statusListIndex.should.be.a('number');
+              issuedVc.credentialStatus.statusListIndex.should.be.gte(0);
+              //   TODO check for base 10
+            });
+          it('The statusListCredential property MUST be a URL to a verifiable credential.',
+            async function () {
+              this.test.cell = { columnId: issuerName, rowId: this.test.title };
+              issuedVc.credentialStatus.should.contain.keys('statusListCredential');
+              issuedVc.credentialStatus.statusListCredential.should.be.a('string');
+              const statusListCredential = (await getSlc({ issuedVc })).slc;
```

Comment:
I have no problem with this, but you could do `const {slc} = await getSlc({issuedVc});`

Totally your choice here.